app/src/sharedTest/java/org/oppia/android/app/databinding/ImageViewBindingAdaptersTest.kt

```diff
@@ -200,14 +198,13 @@ class ImageViewBindingAdaptersTest {
       val url = profileAvatar.avatarImageUri.toString()
       setProfileImage(imageView, profileAvatar)
       testCoroutineDispatchers.runCurrent()
-      lateinit var imageBitmapSource: ImageTarget<Bitmap>
-      testGlideImageLoader.loadBitmap(url, imageBitmapSource)
+      var imageViewBitmap = ImageView(it)
+      testCoroutineDispatchers.runCurrent()
+      testGlideImageLoader.loadBitmap(url, ImageViewTarget(imageViewBitmap))
```

Comment:
So before getting into the test, you'll need to update the adapters class to retrieve ImageLoader from the application context (similar to OppiaClockInjector in TextViewBindingAdapters; the principle is the same).

Then you'll need to replace the Glide code with a call to load the image via ImageLoader, and verify manually that this is working. Once verified, you can use TestGlideImageLoader to retrieve the loaded images (you won't need to call loadBitmap here since the adapter method itself will).